packages/react-components/src/autocomplete/src/Autocomplete.tsx

```diff
@@ -245,16 +264,20 @@ export function InnerAutocomplete(props) {
         // Usually provided by the field inputs.
         "aria-labelledby": ariaLabelledBy,
         "aria-describedby": ariaDescribedBy,
-        menuProps: { id: menuId, style: { width: menuWidth, ...menuStyle } = {}, ...menuProps } = {},
+        menuProps: menuPropsTemp = {}, //TODO: TS i can't find the proper synthax to deconstruct the object here
         as = "input",
         children,
         forwardedRef,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Just like the other PR, size is passe to the Search Input, who doesnt have a size property.

Reply:
i added  omitProps to fieldProps
```
    } = mergeProps(
        props,
        omitProps(fieldProps, ["size"])
    );
```